docs/building/windows-core.md

```diff
@@ -44,10 +44,7 @@ Invoke-WebRequest -Uri https://raw.githubusercontent.com/dotnet/cli/rel/1.0.0/sc
 If you have any problems installing `dotnet`,
 please see their [documentation][cli-docs].
 
-If you are using Windows 7, Windows Server 2008 or Windows Server 2012
-you will also need to install
-[Visual C++ Redistributable for Visual Studio 2012 Update 4][redist-2012]
-and [Visual C++ Redistributable for Visual Studio 2015][redist-2015].
+You will also need to install a recent version of Visual Studio with the optional feature 'Common Tools for Visual C++' installed. The free [Community edition of Visual Studio 2015](https://beta.visualstudio.com/visual-studio-community-vs/) should work fine.
```

Comment:
How about something like, "You will also need to install an edition of Visual Studio 2015 (Community, Enterprise, or Professional) with the optional feature 'Common Tools for Visual C++'"?

The things I thought were missing were the version number of Visual Studio and a list of the supported editions. I am open to the wording of it as long as it incorporates those two details.